lib/Config/PopulatorData/Templates/ModularStyleStories.php

```diff
@@ -150,7 +155,7 @@ private function getBody() {
                                 array (
                                   'type' => 'image',
                                   'link' => '',
-                                  'src' => $this->template_image_url . '/Modular-Logo.png',
+                                  'src' => 'http://mailpoet.info/wp-content/uploads/2017/08/Modular-Logo.png',
```

Comment:
Do we really need to host all of these images on mailpoet.info if we have them on the plugin repo or locally?

Reply:
Good catch! Thank you. I've exported this template several times and I didn't notice this change.